components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -2656,7 +2656,8 @@ private APIKey getSandboxKeyOfApplication(int applicationId, String accessTokenS
                 apiKey.setAccessToken(accessToken);
                 apiKey.setType(resultSet.getString("TOKEN_TYPE"));
                 apiKey.setValidityPeriod(resultSet.getLong("VALIDITY_PERIOD") / 1000);
-
+                apiKey.setGrantTypes(resultSet.getString("GRANT_TYPES"));
```

Comment:
Same string used in Line 2582 as well. Better to move to constant

Reply:
we decided to not to use constants in dao, for the sake of clarity.